src/ovs/aca_ovs_programmer.cpp

```diff
@@ -162,28 +202,38 @@ int ACA_OVS_Programmer::create_update_neighbor_port(const string vpc_id,
     throw std::invalid_argument("tunnel_id is 0");
   }
 
+  overall_rc = setup_ovs_bridges_if_need();
+
+  if (overall_rc != EXIT_SUCCESS) {
+    throw std::runtime_error("Invalid environment with br-int and br-tun");
+  }
+
   string outport_name = aca_get_outport_name(network_type, remote_ip);
 
   string cmd_string =
-          "add-port br-tun " + outport_name + " -- set interface " +
+          "--may-exist add-port br-tun " + outport_name + " -- set interface " +
```

Comment:
--may-exist? ....

Reply:
yes, this would fix all the create_update_neighbor error we are seeing in our integration today.

The situation is:
* Host 1 receives P3,P4 neighbor info, both P3,P4 is hosted on Host 2
* When host 1 process P3 neighbor, it creates the outport which maps to host 2
* When host 1 process P4 neighbor, it creates the same outport for host 2, therefore "--may-exist" is needed to avoid error